internal/pkg/vm/state/testing.go

```diff
@@ -8,138 +8,48 @@ import (
 	"github.com/filecoin-project/go-address"
 	"github.com/ipfs/go-cid"
 	cbor "github.com/ipfs/go-ipld-cbor"
-	"github.com/stretchr/testify/mock"
 	"github.com/stretchr/testify/require"
 
 	"github.com/filecoin-project/go-filecoin/internal/pkg/vm/actor"
-	"github.com/filecoin-project/go-filecoin/internal/pkg/vm/internal/dispatch"
 )
 
-// MustFlush flushes the StateTree or panics if it can't.
-func MustFlush(st Tree) cid.Cid {
-	cid, err := st.Flush(context.Background())
+// NewFromString sets a state tree based on an int.  TODO: this indirection
+// can be avoided when we are able to change cborStore to an interface and then
+// making a test implementation of the cbor store that can map test cids to test
+// states.
```

Comment:
It is an interface now, so maybe update to "We could avoid this indirection by ..."